nflow-engine/src/main/java/io/nflow/engine/config/db/MysqlDatabaseConfiguration.java

```diff
@@ -45,8 +45,9 @@ public MysqlDatabaseConfiguration() {
    */
   @Bean
   @Override
-  @SuppressFBWarnings(value = { "CLI_CONSTANT_LIST_INDEX", "WEM_WEAK_EXCEPTION_MESSAGING" },
-      justification = "extracting major and minor version from splitted string, exception message is ok")
+  @SuppressFBWarnings(
+      value = { "CLI_CONSTANT_LIST_INDEX", "WEM_WEAK_EXCEPTION_MESSAGING", "RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE" },
```

Comment:
Is CLI_CONSTANT_LIST_INDEX required? I do not see it used here

Reply:
not anymore, removed